facebook.js

```diff
@@ -14,15 +14,17 @@ const emoji_map = {
   "car": "🚗",
   "child": "👦",
   "christmas tree": "🎄",
-  "closeup": "👀",
```

Comment:
Only saw the other one, maybe both are existing, I updated the PR keeping the two spellings

![capture d ecran 2017-01-02 a 22 15 01](https://cloud.githubusercontent.com/assets/2368736/21596267/4fc278be-d139-11e6-992d-9a3d33f03da4.png)